libevm/tooling/release/cherrypick.sh

```diff
@@ -44,4 +44,4 @@ if [[ -z "${commits[*]// }" ]]; then # $x// removes whitespace
     exit 0;
 fi
 
-git cherry-pick "${commits[@]}";
+git cherry-pick -S "${commits[@]}";
```

Comment:
I have my git config set up to always sign, but it seems git cherry-pick does not sign commits after resolving conflicts + `git cherry-pick --continue` 🤷